batch/batch/worker/worker.py

```diff
@@ -1789,6 +1796,323 @@ def __str__(self):
         )
 
 
+def write_int(writer: asyncio.StreamWriter, v: int):
```

Comment:
Is there anyway to move all of this new code into another file in the module? I already didn't like how long this file was getting.